### DISCUSSION TITLE
Ask? (graphical apps)

### DISCUSSION THREAD

**Discussion by rsanjuan87** [lima-vm/lima#1473] [Q&A] [answered]
Its posible run graphical apps ? ....and most important in my case its posible run WayDroid?

**Comment by afbjorklund** [comment, marked as answer]
It is possible to run graphical apps, the main requirement is to enable a display.

```yaml
video:
  display: "default"
```

* https://github.com/lima-vm/lima/issues/989

----

For waydroid specifically, you also need to install wayland - easiest is with gdm3:

https://linuxconfig.org/how-to-enable-disable-wayland-on-ubuntu-22-04-desktop

You need to install `gdm3`, change the config, and add a password for the VM user.

**Reply by afbjorklund** [reply]
Here is a provision script to install the `waydroid` program, in an ubuntu-lts instance.

```yaml
provision:
- mode: system
  script: |
    #!/bin/bash
    set -eux -o pipefail
    command -v waydroid >/dev/null 2>&1 && exit 0
    sudo apt install linux-modules-extra-$(uname -r) -y
    # https://docs.waydro.id/usage/install-on-desktops#ubuntu-debian-and-derivatives
    sudo apt install curl ca-certificates -y
    curl https://repo.waydro.id | sudo bash
    sudo apt install waydroid -y
```

![lima-waydroid-session-start](https://user-images.githubusercontent.com/10364051/231004241-647a1907-6e0e-4998-a910-109dc6f85678.png)

**Reply by rsanjuan87** [reply]
I´m facing the problem that no mouse cursor 
I tried including `,show-cursor=on` after default but nothing happend
> It is possible to run graphical apps, the main requirement is to enable a display.
> 
> ```yaml
> video:
>   display: "default"
> ```
>

**Reply by afbjorklund** [reply]
Which platform ? There might be some trick needed, for software cursor...

EDIT: But only know how to do it for Xorg, not for Wayland... (still too new)
